### DISCUSSION TITLE
如何自定义help菜单？

### DISCUSSION THREAD

**Discussion by ABT98421** [Arondight/Adachi-BOT#491] [Q&A] [answered]
找不到./src/plugins/tools/help.js

**Comment by mark9804** [comment]
看看 #295 的讨论有没有帮助？

**Reply by Arondight** [reply]
这个过时了

**Comment by Arondight** [comment, marked as answer]
在这儿，换成自己的就行

https://github.com/Arondight/Adachi-BOT/blob/d7995e7c4cd2114aec9030c974c8b2bdaa9f3243/src/plugins/tools/index.js#L36-L38

**Reply by ABT98421** [reply]
> 在这儿，换成自己的就行
> 
> https://github.com/Arondight/Adachi-BOT/blob/d7995e7c4cd2114aec9030c974c8b2bdaa9f3243/src/plugins/tools/index.js#L36-L38

没看懂这里要怎么改，再细说细说？

**Reply by Arondight** [reply]
> 没看懂这里要怎么改

```
 case hasEntrance(msg.text, "tools", "help"):  {
   const text = ```这是我自己的帮助菜单
1. 干这个
2. 干那个
```;
   msg.bot.say(msg.sid, text, msg.type, msg.uid, false, "\n"); 
   break; 
  }
```

**Reply by ABT98421** [reply]
现在这样了
![image](https://user-images.githubusercontent.com/78371203/146154424-e1b5a197-25fb-41f7-8efa-9eeb815d1526.png)

**Reply by ABT98421** [reply]
把```改成`就好了，ok了

**Reply by Arondight** [reply]
不好意思，上班时候看这个，语言串台了
